src/main/java/com/gb/cornucopia/CornuCopia.java

```diff
@@ -19,75 +12,41 @@
 
 
 @Mod(modid = CornuCopia.MODID, version = CornuCopia.VERSION)
-public class CornuCopia
-{
-	public static final String NAME = "CornuCopia";
-	public static final String MODID = "cornucopia";
-	public static final String VERSION = "1.2.5";
-
-	@Instance(value = CornuCopia.MODID)
-	public static CornuCopia instance;
-	public static Settings config;
-	
-	@SidedProxy(clientSide="com.gb.cornucopia.ClientProxy", serverSide="com.gb.cornucopia.ServerProxy")
-	public static ServerProxy proxy;
-
-	//region creative tabs
-	public static final CreativeTabs tabBees = new CreativeTabs("cc_bees"){
-		@Override
-		public Item getTabIconItem() {
-			return Bees.queen;
-		}    	
-	};
-	public static final CreativeTabs tabVeggies = new CreativeTabs("cc_veggies"){
-		@Override
-		public Item getTabIconItem() {
-			return Veggie.asparagus.raw;
-		}
-	};
-	public static final CreativeTabs tabFruit = new CreativeTabs("cc_fruit"){
-		@Override
-		public Item getTabIconItem() {
-			return Fruit.peach.raw;
-		}
-	};    
-	public static final CreativeTabs tabCookery = new CreativeTabs("cc_cookery"){
-		@Override
-		public Item getTabIconItem() {
-			return Item.getItemFromBlock(Cookery.cutting_board);
-		}
-	};
-	public static final CreativeTabs tabCuisine = new CreativeTabs("cc_cuisine"){
-		@Override
-		public Item getTabIconItem() {
-			return Cuisine.bread_dough;
-		}
-	};
-
-	//endregion
-
-	@EventHandler
-	public void preInit(final FMLPreInitializationEvent e)
-	{
-		CornuCopia.proxy.preInit(e);
-	}
-
-	@EventHandler
-	public void init(final FMLInitializationEvent e)
-	{
-		CornuCopia.proxy.init(e);
-		// no proxy needed, this hooks up containers between client and server. misleading name
-		GuiHandler.init(this);
-	}
-
-	@EventHandler
-	public void PostInit(final FMLPostInitializationEvent e)
-	{
-		CornuCopia.proxy.postInit(e);
-	}
-
-	@EventHandler
-	public void serverLoad(final FMLServerStartingEvent event)
-	{
-	}
-}
+public class CornuCopia {
+    public static final String NAME = "CornuCopia";
+    public static final String MODID = "cornucopia";
+    public static final String VERSION = "1.2.5";
+
+    @Mod.Instance(value = CornuCopia.MODID)
+    public static CornuCopia instance;
+    public static Settings config;
+
+    @SidedProxy(clientSide = "com.gb.cornucopia.proxy.ClientProxy", serverSide = "com.gb.cornucopia.proxy.ServerProxy")
```

Comment:
`clientSide = "com.gb.cornucopia.proxy.ClientProxy"` this would fail now that `ClientProxy` has been removed right? AFAIK that would be a runtime thing if you haven't seen it